### DISCUSSION TITLE
Missing features in the 1.19.2 version

### DISCUSSION THREAD

**Discussion by IroPlisk** [MrTJP/ProjectRed#1889] [Q&A]
I remember asking this but the discussion had gone cold and was eventually buried. But where are the solar panels in the 1.19.2 latest release? and what about the TODO on the IC workbench's "author" field? i also clearly remember that in 1.7.10, the IC workbench had a much wider area to work with, but it seems that in this version there is only one default schematic size. 

My question is thus, are these features ever going to be re-implemented? i feel like i missed some, and i most surely did, but these felt most important, at least to me.

**Comment by TahsinCr** [comment]
I'm experiencing the same issue, especially when it comes to resizing.
